code/datums/keybinding/living.dm

```diff
@@ -152,3 +152,42 @@
 	var/mob/living/M = user.mob
 	M.toggle_move_intent()
 	return TRUE
+
+
+/datum/keybinding/living/item_pixel_shift
+	hotkey_keys = list("V")
+	name = "item_pixel_shift"
+	full_name = "Item Pixel Shift"
+	description = "Shift a pulled item's offset"
+	category = CATEGORY_MISC
+	keybind_signal = COMSIG_KB_LIVING_ITEM_PIXEL_SHIFT_DOWN
+
+/datum/keybinding/living/item_pixel_shift/down(client/user)
+	. = ..()
+	if(.)
+		return
+	user.mob.AddComponent(/datum/component/pixel_shift)
+	SEND_SIGNAL(user.mob, COMSIG_KB_LIVING_ITEM_PIXEL_SHIFT_DOWN)
```

Comment:
This isn't really how we should use signals; this is just a proc call and not an event.
It also isn't clear to me why this is a component in the first place though to be honest.